### PR TITLE
Add 'Still Interested?' email

### PR DESCRIPTION
**Pull request by sedders123**
9th of December was an arbitrary deadline, happy to change it to something else. 

We should attach the kit disclaimer form to this email when sending.

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
Would it be worth me pulling the data regarding IDE activity and user accounts as a feed-in to this?

**Comment by PeterJCLaw** [comment]
I have pulled said data, it's in Slack, in the [#teams channel](https://studentrobotics.slack.com/messages/teams).

File: SR2020/2019-11-29-still-interested.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+to: Teams that haven't responded to any emails
+subject: Still interested in SR2020?
+---
+
+Hello!
+
+Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll allocate your kit to another team.
```
Comment (RealOrangeOne) [inline SR2020/2019-11-29-still-interested.md]: Maybe worth making it seem less forceful? 

```suggestion
Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, ideally with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll be forced allocate your kit to another team.
```

Comment (trickeydan) [reply SR2020/2019-11-29-still-interested.md]:
```suggestion
Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, ideally with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll be forced to allocate your kit to another team.
```

Comment (sedders123) [reply SR2020/2019-11-29-still-interested.md]: With regards to `ideally` is it really worth adding? If they don't send one we'll just need to send another email asking for it.

**Comment by PeterJCLaw** [comment]
This is now going to go out after #23, so we should review this with that in mind.

**Comment by RealOrangeOne** [comment]
Sent to the 3 teams

**Pull request merged**